pkg/openstack/tasks/floating_ips.go

```diff
@@ -239,9 +231,7 @@ func collectFloatingIPs(ctx context.Context, payload CollectFloatingIPsPayload)
 
 	logger.Info(
 		"populated openstack floating IPs",
-		"project_id", projectID,
-		"region", region,
-		"domain", domain,
+		"scope", payload.Scope,
```

Comment:
I suppose the resulting log event won't be structured at all ? Can we improve that, so that we can easily read/parse/lookup these as well?